manifests/mod/fastcgi.pp

```diff
@@ -1,6 +1,8 @@
 class apache::mod::fastcgi {
   include ::apache
-
+  if ($::osfamily == 'Redhat' and $::operatingsystemrelease > '6') {
+    fail("mod_fastcgi is no longer supported on el7 and above.")
```

Comment:
I do wonder if other distros will deprecate it too and if there should be a more generalized check but I think this could be fine for now and refactored later.